src/current/v24.1/ui-physical-cluster-replication-dashboard.md

```diff
@@ -49,6 +45,17 @@ Hovering over the graph displays:
 - The date and time.
 - The number of SST bytes replicated.
 
+## Replication lag
+
+<img src="{{ 'images/v24.1/ui-replication-lag.png' | relative_url }}" alt="DB Console Replication Lag graph showing results over the past hour" style="border:1px solid #eee;max-width:100%" />
+
+{% include_cached new-in.html version="v24.1" %} The **Replication Lag** graph displays the [replication lag]({% link {{ page.version.version }}/physical-cluster-replication-technical-overview.md %}) between the primary and standby cluster. This is the time between the most update-to-date replicated time and the actual time.
```

Comment:
> This is the time between the most update-to-date replicated time and the actual time.

I think this is a typo — should be `up-to-date replicated time`?

Reply:
Yes, corrected, thanks!